conbench/templates/app.html

```diff
@@ -7,78 +7,74 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 {% block styles %}
 <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
-<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
-<link href="https://cdn.datatables.net/1.10.23/css/dataTables.bootstrap.min.css" rel="stylesheet" type="text/css">
-<link href="https://cdn.datatables.net/responsive/2.2.7/css/responsive.bootstrap.min.css" rel="stylesheet" type="text/css">
-<link href="{{ url_for('static', filename='app.css') }}" rel=stylesheet type=text/css />
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+<link href="https://cdn.datatables.net/v/bs5/dt-1.13.3/r-2.4.0/datatables.min.css" rel="stylesheet">
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+<link href="{{ url_for('static', filename='app.css') }}" rel=stylesheet type=text/css>
```

Comment:
The meat of the change that provoked all the breakages.

Note that previously we explicitly included `https://cdn.datatables.net/responsive/2.2.7/css/responsive.bootstrap.min.css`

Now that is also included in a combined CSS resources. `dt-1.13.3/r-2.4.0/datatables.min.css` means: datatables version 1.13.3 *plus* responsive 2.4.0. Note: I don't think we make proper use of this 'responsive' CSS plugin; will review later.

Reply:
In addition we now include
```
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
```
because the bootstrap 3 glyphicons do not exist anymore; bootstrap icons now is a separate project.